_locales/fr/messages.json

```diff
@@ -1044,12 +1050,18 @@
     "receiverRcInterpolationInterval": {
         "message": "Intervalle d'interpolation RC [ms]"
     },
+    "pidTuningDtermSetpointTransition": {
+        "message": "D Setpoint transition"
+    },
     "pidTuningPtermSetpoint": {
         "message": "D Setpoint transition"
     },
     "pidTuningDtermSetpoint": {
         "message": "D Setpoint Weight"
     },
+    "pidTuningDtermSetpointTransitionHelp": {
+        "message": "Avec ce param\u00e8tre, le D Setpoint Weight peut \u00eatre r\u00e9duit sur le centre des sticks, r\u00e9sultant sur des flips et des rolls plus doux en fin de mouvement. <br> La valeur repr\u00e9sente le point de d\u00e9viation du stick: 0 - stick centr\u00e9, 1 - d\u00e9viation maximale. Lorsque le stick est au-dessus de ce point, le Setpoint Weight est constamment \u00e0 la valeur configur\u00e9e. Lorsque le stick est en-dessous de ce point, le Setpoint Weight est proportionnellement r\u00e9duit atteignant 0 sticks centr\u00e9s.<br> La valeur 1 donnera un effet maximal de douceur, tandis que la valeur 0 gardera le Setpoint Weight sur toute la plage des sticks \u00e0 la valeur configur\u00e9e."
+    },
     "pidTuningPtermSetpointHelp": {
```

Comment:
This should be removed, it's not used any more, and was removed from the English file.